src/Voron/Impl/SliceSmallSet.cs

```diff
@@ -220,16 +223,12 @@ public void Dispose()
 
         public void Remove(Slice name)
         {
-            if (_overflowStorage != null)
-            {
-                _overflowStorage.Remove(name);
-                return;
-            }
+            _overflowStorage?.Remove(name);
 
             int idx = FindKey(name);
```

Comment:
It can return `Invalid`, can't it? Unless I'm missing something it might result in accessing below arrays on `-1` position.

Reply:
Yes, good catch.